build-daq.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+# we can't use safe mode, shitty scripts being called
+# set -o errexit -o nounset -o pipefail
+shopt -s expand_aliases # script uses aliases, does't work in non-interactive shells unless explicitly enabled
+# shitty scripts
+# IFS=$'\n\t\v'
+cd `dirname "${BASH_SOURCE[0]:-$0}"`
+
+# Constructed from
+# https://github.com/DUNE-DAQ/minidaqapp/wiki/Instructions-for-setting-up-a-v2.4.0-development-environment
+
+echo "starting setup"
+
+# tmp_dir=$(mktemp -d -t ci-XXXXXXXXXX)
```

Comment:
This must be user-configurable, ideally with a command line argument or at least target `$PWD`.
`tmp/` is not a great choice, there's a risk of clash if multiple users run on `tmp/${USER}` is better option.